filament/include/filament/Renderer.h

```diff
@@ -136,6 +136,20 @@ class UTILS_PUBLIC Renderer : public FilamentAPI {
      */
     void render(View const* view);
 
+    /**
+     * Mirror the currently rendered view to the indicated swap chain, using the
+     * indicated source and destination rectangle.
+     *
+     * @param toSwapChain The swap chain into which the frame should be mirrored.
+     * @param toView The destination rectangle in which to draw the view.
+     * @param fromView The source rectangle to be mirrored.
+     *
+     * @remark
+     * mirrorFrame() should be called after a frme is rendered using render()
+     * but before endFrame() is called.
+     */
+    void mirrorFrame(SwapChain* toSwapChain, View const* toView, View const* fromView);
```

Comment:
The parameters shouldn't be `View*` but `Viewport*`. Also use `dst` and `src`.

Reply:
done.